applications/segyio-catr.c

```diff
@@ -474,6 +474,7 @@ int main( int argc, char** argv ) {
         default: labels = su; break;
     }
 
+    /* verify all ranges are proper */
```

Comment:
Should this end in something like "..rly defined"..?

Reply:
I suppose that would be better English, sure.

Reply:
The ranges would always be defined, but they might be horribly broken.